Lib/glyphsLib/util.py

```diff
@@ -130,6 +130,14 @@ def pairs(list):
     return [list[i : i + 2] for i in range(0, len(list), 2)]
 
 
+def freezedict(dct):
+    return frozenset(dct.items())
+
+
+def unfreezedict(frozen):
+    return dict(list(frozen))
```

Comment:
I don't think you need this `unfreezedict` helper at all, `dict(frozen)` already does what you want, no? `dict` takes an iterable already, don't need to make it a `list` first.